crates/rpc/rpc/src/eth/api/fee_history.rs

```diff
@@ -342,6 +348,9 @@ impl FeeHistoryEntry {
         FeeHistoryEntry {
             base_fee_per_gas: block.base_fee_per_gas.unwrap_or_default(),
             gas_used_ratio: block.gas_used as f64 / block.gas_limit as f64,
+            base_fee_per_blob_gas: block.blob_fee(),
+            blob_gas_used_ratio: block.blob_gas_used() as f64 /
+                reth_primitives::constants::eip4844::MAX_DATA_GAS_PER_BLOCK as f64,
```

Comment:
this looks correct to me

Reply:
used as a reference:
https://github.com/NethermindEth/nethermind/blob/d6bf96493c11a487e0af69b2c6fba323d1216b05/src/Nethermind/Nethermind.JsonRpc/Modules/Eth/FeeHistory/FeeHistoryOracle.cs#L64